src/pyocr/builders.py

```diff
@@ -53,15 +53,17 @@ def __init__(self, content, position):
         content = to_unicode(content)
         self.content = content
         self.position = position
+        self.confidence = confidence
 
     def get_unicode_string(self):
         """
         Return the string corresponding to the box, in unicode (utf8).
         This string can be stored in a file as-is (see write_box_file())
         and reread using read_box_file().
         """
-        return to_unicode("%s %d %d %d %d") % (
+        return to_unicode("%s %s %d %d %d %d") % (
```

Comment:
As is, it will break `tesseract.CharBoxBuilder`.
`CharBoxBuilder` correspond to a file format specific to Tesseract (configuration 'makebox'). If you modify this function, the format won't be the same than Tesseract anymore, and `CharBoxBuilder.read_file()` won't be able to read files written by `CharBoxBuilder.write_file()` anymore.

Reply:
Reverted changes